SU2_CFD/src/solver_direct_turbulent.cpp

```diff
@@ -3271,6 +3305,8 @@ void CTurbSSTSolver::BC_Outlet(CGeometry *geometry, CSolver **solver_container,
 void CTurbSSTSolver::BC_Fluid_Interface(CGeometry *geometry, CSolver **solver_container, CNumerics *conv_numerics,
     CNumerics *visc_numerics, CConfig *config){
 
+///////////// This has to be changed for Rinaldi approach
+
```

Comment:
Here it just works for one vertex, if I am correct. Maybe we can add an error for the moment when used with the approach of Enrico? What would be its behavior, with this current implementation?